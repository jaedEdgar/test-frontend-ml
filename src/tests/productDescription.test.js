import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import ProductDescription from "../client/components/productDescription"
import '@testing-library/jest-dom'

test('render ItemDescription text validate ', () => {
  render(<ProductDescription description={'Lorem ipsum'} />);
  const text = screen.getByText(/Lorem ipsum/i);
  const title = screen.getByText(/DescripciÃ³n del producto/i);
  expect(text).toBeInTheDocument()
  expect(title).toBeInTheDocument()
});


test('should render my component', () => {
  const component = renderer.create(<ProductDescription description={'Lorem ipsum'} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
