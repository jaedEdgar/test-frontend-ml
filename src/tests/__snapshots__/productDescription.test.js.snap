// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render my component 1`] = `
<div
  className="container"
>
  <div
    className="product-description"
  >
    <span
      className="product-description--header"
    >
      Descripci√≥n del producto
    </span>
    <p
      className="product-description--paragraph"
    >
      Lorem ipsum
    </p>
  </div>
</div>
`;
